source/includes/candidate-ingestion/_current_user.md

```diff
@@ -28,8 +30,10 @@ This request is simply the URL. It takes no parameters.
 
 ### Response Parameters
 
-Property Name | Type | Required
--------------- | -------------- | -------------- 
-first_name | String | Yes
-last_name | String | Yes
-email | String | Yes
\ No newline at end of file
+Property Name | Type | Required | Description
```

Comment:
Is `required` a meaningful column on a read-only endpoint?

Reply:
that's a good point! i shall remove it

Reply:
it's funny it's listed in so many other places for Get request, should i remove them all?